src/test/java/org/folio/rest/api/RequestBatchAPITest.java

```diff
@@ -245,13 +247,15 @@ public void shouldPublishKafkaEventWhenUpdateRequestPositionsInBatchForTheInstan
       .map(JsonObject.class::cast)
       .sorted(Comparator.comparingInt(obj -> obj.getInteger("position")))
       .toList();
+    String firstRequestId = firstRequest.getString("id");
+    String secondRequestId = secondRequest.getString("id");
     assertThat(requestsSorted.get(0).getInteger("position"), is(1));
     assertThat(requestsSorted.get(1).getInteger("position"), is(2));
-    assertThat(requestsSorted.get(0).getString("id"), is(secondRequest.getString("id")));
-    assertThat(requestsSorted.get(1).getString("id"), is(firstRequest.getString("id")));
+    assertThat(requestsSorted.get(0).getString("id"), is(secondRequestId));
+    assertThat(requestsSorted.get(1).getString("id"), is(firstRequestId));
 
-    assertRequestQueueReorderingEvent(instanceId.toString(), List.of(
-      firstRequest.getString("id"), secondRequest.getString("id")));
+    assertRequestQueueReorderingEvent(instanceId.toString(),
+      requestsSorted.get(1).getString("itemId"), List.of(firstRequestId, secondRequestId), TITLE);
```

Comment:
m.b. it's a good idea to extract constants ?

Reply:
assertRequestQueueReorderingEvent is a universal method that is also applicable to queues with ITEM requestLevel requests